packages/common/src/store/reducers.ts

```diff
@@ -263,6 +268,8 @@ export type CommonState = {
   tracks: TracksCacheState
   users: UsersCacheState
 
+  savedCollections: SavedCollectionsState
```

Comment:
nit: IMO a bit cleaner to use the `ReturnType` of the reducer here, and then we don't have to export the state